dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -3107,6 +3107,46 @@ void Sedp::data_received(DCPS::MessageId message_id,
 }
 #endif
 
+//This is a function to unify the notifcation of liveliness within RTPS
+//The local participant map is checked for associated entities and then they are notified
+//of liveliness if their QoS is compatibile
```

Comment:
Please move this to the header and make it a doxygen comment, which use `/// ` or `/** `.